examples/paywall-tester/src/main/java/com/revenuecat/paywallstester/PaywallTesterApp.kt

```diff
@@ -48,8 +55,18 @@ private fun AppNavHost(
             PaywallScreen()
         }
         composable(
-            route = AppScreen.PaywallFooter.route.plus("/{${PaywallScreenViewModel.OFFERING_ID_KEY}}"),
-            arguments = listOf(navArgument(PaywallScreenViewModel.OFFERING_ID_KEY) { type = NavType.StringType }),
+            route = AppScreen.PaywallFooter.route
+                .plus("/{${PaywallScreenViewModel.OFFERING_ID_KEY}}")
+                .plus(
+                    "?${PaywallScreenViewModel.FOOTER_CONDENSED_KEY}={${PaywallScreenViewModel.FOOTER_CONDENSED_KEY}}",
```

Comment:
Actually, this is defining the route, not an actual value. Basically since `{${PaywallScreenViewModel.FOOTER_CONDENSED_KEY}}` is between curly braces, it means it's a parameter that can be obtained later programmatically. It's a bit confusing since it has the same name as the query parameter, but I think it's correct right now.

Reply:
Ooooh that makes sense.